collectors/0/samza_staging.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+import json
+import re
+import sys
+
+from kafka import KafkaConsumer
```

Comment:
I assume `kafka-python` is already installed:

https://github.com/optimizely/backend-chef/blob/8a7d52cb001bf13ec553a2337d6c8dc5209154ae/cookbooks/optimizely_tcollector/recipes/install_python_packages.rb#L23